sdk/src/types/block/output/metadata.rs

```diff
@@ -17,21 +17,11 @@ pub struct OutputMetadata {
     output_id: OutputId,
     /// Whether the output is spent or not.
     is_spent: bool,
-    /// If spent, the index of the milestone in which the output was spent.
+    /// Commitment ID that includes the output.
```

Comment:
we seem to be using a mix of ID and identifier.

Reply:
Done